### ISSUE TITLE
Improving accessibility

### ISSUE THREAD

**Issue by nciske** [cimburadotcom/salesforce-wordpress-to-lead#69]
The forms generated by the plugin are currently failing WCAG 2.0 checks, which is especially important for government agencies. We think it should be a fairly easy fix. For accessibility reasons all form elements should be labelled as well as providing a placeholder option where appropriate. Styles should be provided to allow users to hide labels except from screen readers. The exact WCAG 2.0 guideline it's failing at the moment is guideline 3.3.2 Labels or Instructions. 

So instead of making labels and placeholders and either/or option it would be good to generate labels for all form elements, including hidden ones, and add some hidden styling for them. The option then changes to "Show labels for visible elements or use placeholders." Otherwise it's a fantastic plugin, very easy to use.

Topic Link: https://wordpress.org/support/topic/improving-accessibility-2/
